charts/datarepo/Chart.yaml

```diff
@@ -47,6 +47,6 @@ dependencies:
   repository: https://broadinstitute.github.io/datarepo-helm/
   condition: create-secret-manager-secret.enabled
 - name: de-elasticsearch
-  version: 0.1.3
+  version: 0.1.4
```

Comment:
I think the overall chart version need to be bumped too, no?

Reply:
I'm not sure so I created a thread internally.